runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -727,7 +727,7 @@ private boolean isLogDataValid(long address, ILogData logData, boolean throwExce
 
         if (logData.isTrimmed()) {
             if (throwException) {
-                throw new TrimmedException(String.format("Trimmed address %s", address));
+                throw new TrimmedException(Collections.singletonList(address));
```

Comment:
Cleaner to push down to the constructor.

Reply:
Done.